frontend/src/pages/docs/[...slug].tsx

```diff
@@ -275,7 +279,7 @@ const StyledLeftNav = styled.div`
   ::-webkit-scrollbar-thumb {
     background-clip: padding-box;
     border-right: 4px #f8f8f8 solid;
-    background: grey;
+    background-color: grey;
```

Comment:
gray vs grey? are both recognized?

Reply:
ooh never thought about that 💡 !

Just confirmed that they are both recognized!

https://www.w3schools.com/colors/colors_names.asp

<img width="1286" alt="Screen Shot 2022-08-24 at 11 07 21 AM" src="https://user-images.githubusercontent.com/6309723/186491537-ba277cd7-2fac-48b6-bdd9-e7f1ff7b48f1.png">